### PR TITLE
feat: normalize redux state with short tokens

### LINKED ISSUE #2353: Use short tokens in redux normalized state.
While investigating #2351 I found that the cause is actually inconsistent normalized redux state, where we use 
full tokens when initially fetching the proposal but after a successful edit we update the redux store using short tokens.

We should use short tokens in our normalized redux store to ease the lookup for both short & full tokens.

### PR DESCRIPTION
**Pull request by thi4go**
This diff normalizes our entire redux state to use short tokens instead of full tokens, resolving various bugs that may happen when the app expects one token form, and receives another.

closes #2353

### CODE REVIEW

**Comment by amassarwi** [comment]
@thi4go thanks for this mate, this should be closing #2351 I think.

**Comment by tiagoalvesdulce** [comment]
We should wait for e2e to be ready before getting this in since it touches a lot of sensitive parts of the app

**Comment by tiagoalvesdulce** [comment]
Closed by mistake

**Comment by thi4go** [comment]
@tiagoalvesdulce e2e already being very useful. caught some minor bug that could otherwise pass unoticed on a review 👍 :)

this PR is ready

**Comment by tiagoalvesdulce** [comment]
Nice! Can you run `yarn prettify`?

**Review by tiagoalvesdulce — CHANGES_REQUESTED** [review]
I ran prettify for thiago because his dev env was messed up.

While testing this I found several Issues. See:

Duplicated new proposal and proposal not showing on top of discussion tab:

https://user-images.githubusercontent.com/13955303/118496651-b3d39800-b6fa-11eb-86b2-5967042372cf.mp4

Wrong tabs:
Screen Shot 2021-05-17 at 10 24 31
Screen Shot 2021-05-17 at 10 25 01
Screen Shot 2021-05-17 at 10 25 18
Screen Shot 2021-05-17 at 10 25 25

Voting proposal showing up at admin's unreviewed tab:

Screen Shot 2021-05-17 at 10 26 06

**Comment by thi4go** [comment]
Thanks for the review @tiagoalvesdulce, nice catch. Will be fixing this asap

**Comment by tiagoalvesdulce** [comment]
I think the sorting is still messed up in the "In Discussion" tab. I don't know if it's intentional but it's putting proposals with voting authorized on top of others:

Screen Shot 2021-05-25 at 18 30 38

The proposal "sssssss" was created after the proposal "oooooooooo".

File: src/components/RecordsView/RecordsView.jsx [hunk 1]
```diff
@@ -105,6 +108,8 @@ const RecordsView = ({
     [loadingItems, placeholder]
   );
 
+  console.log(filteredRecords);
```
Comment (tiagoalvesdulce) [inline src/components/RecordsView/RecordsView.jsx:111]: log

**Review by tiagoalvesdulce — CHANGES_REQUESTED** [review]
Sorting is working as expected now.

Seeing this error when trying to submit an edited proposal:

Screen Shot 2021-05-26 at 10 21 48

Is the endpoint expecting the full token?

**Review by tiagoalvesdulce — APPROVED** [review]
LGTM

**Pull request merged**